tests/snapshots/semgrep-core/84cced48c6d1/stdxxx

```diff
@@ -0,0 +1,121 @@
+Snapshot for Test_scan_subcommand.test_basic_verbose_output.(fun)
+--- begin input files ---
+rules.yml
+stupid.py
+--- end input files ---
+[<MASKED TIMESTAMP>][INFO]: Running external command: 'git' 'init' '-b' 'main'
+Initialized empty Git repository in <TMP>/<MASKED>/.git/
+[<MASKED TIMESTAMP>][INFO]: Running external command: 'git' 'config' 'user.name' 'Tester'
+[<MASKED TIMESTAMP>][INFO]: Running external command: 'git' 'config' 'user.email' 'tester@example.com'
+[<MASKED TIMESTAMP>][INFO]: Running external command: 'git' 'add' '.'
+[<MASKED TIMESTAMP>][INFO]: Running external command: 'git' 'commit' '-m' 'Add files'
+[main (root-commit) <MASKED>] Add files
+ 2 files changed, 11 insertions(+)
+ create mode 100644 rules.yml
+ create mode 100644 stupid.py
+[<MASKED TIMESTAMP>][INFO]: Semgrep version: 1.69.0
```

Comment:
still need to mask the semgrep version!!
otherwise next release the test will fail ...